### ISSUE TITLE
helmsman latest release is not compatible with helm 3.3.2

### ISSUE THREAD

**Issue by ndegory** [Praqma/helmsman#513]
Helm 3.3.2 introduced [a breaking change](https://github.com/helm/helm/commit/882eeac7271858124a3cecbe22c5d7d61560714f) (on a patch release), helm repo add will fail if the repo has already been added.
This is something that happens often with Helmsman, running the tests on master with this version of Helm now fails:

```
❯ make build test
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/ndegory/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /Users/ndegory/.kube/config
Error: repository name (stable) already exists, please specify a different name
make: *** [repo] Error 1
```

**Comment by mkubaczyk** [comment]
We're actually checking if specific repository is already there with [helm_helpers.go#L137](https://github.com/Praqma/helmsman/blob/master/internal/app/helm_helpers.go#L137). We might need more data to find out what's wrong like your DSF file and an output of your helm repo list command.

**Comment by ndegory** [comment]
@mkubaczyk I agree, the code looks like it shouldn't try to add the repo if it's already there. The only issue is in the Makefile, which adds the stable repo, this once can easily be fixed by adding the --force-update option.

**Comment by antoinedeschenes** [comment]
helm 3.3.3 was released as a hotfix for a chart repository file problem 

Indeed a fix is required, since helmsman tries to update repos as soon as the url is not identical (including basic auth credentials without #505)

Code could either call `--force-update` or skip existing repos by default

**Comment by ndegory** [comment]
A systematic `--force-update` would defeat the original intent of that change in Helm. Also, it's a new option, I'm not sure we can assume the Helm CLI available in the path is at least 3.3.2.
Wouldn't new settings for the `helm repo add` command behavior be a better option? Something like settings.repositories.forceUpdate and settings.repositories.skipIfExists both defaulting to false to keep the current behaviour.

**Comment by mkubaczyk** [comment]
Ok, I see. I just went for --force-update flag, but you're right. Before 3.3.2 there's no such flag, so we need to distinguish the cases before and after.
